src/ig/IGExporter.ts

```diff
@@ -180,7 +181,7 @@ export class IGExporter {
    *
    * @param igPath {string} - the path where the IG is exported to
    */
-  private addIndex(igPath: string) {
+  private addIndex(igPath: string): void {
```

Comment:
Oh, look at you adding return types!  Thanks for cleaning up after my slopping coding! ;-)

Reply:
Well some of them were my sloppy coding ;)